### PR TITLE
pinning ohai, eph_lvm, rightscale_tag, machine_tag

### PR DESCRIPTION
**Pull request by rshade**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -18,6 +18,9 @@
 depends 'rightscale_backup', '~> 1.2.0'
 depends 'dns', '~> 0.1.3'
 depends 'git', '~> 4.0.2'
+depends 'aws', '~> 2.9.3'
```
Comment (cdwilhelm) [inline metadata.rb]: why are you depending on  aws?

File: Berksfile [hunk 2]
```diff
@@ -5,14 +5,15 @@ metadata
...
-cookbook 'rightscale_volume',github: 'rightscale-cookbooks/rightscale_volume'
-cookbook 'rightscale_tag',github: 'rightscale-cookbooks/rightscale_tag'
-cookbook 'machine_tag',github: 'rightscale-cookbooks/machine_tag'
-cookbook 'ephemeral_lvm',github:'rightscale-cookbooks/ephemeral_lvm'
+cookbook 'rightscale_backup',github: 'rightscale-cookbooks/rightscale_backup', branch: 'v1.2.0'
+cookbook 'rightscale_volume',github: 'rightscale-cookbooks/rightscale_volume', branch: 'v1.3.0'
+cookbook 'rightscale_tag',github: 'rightscale-cookbooks/rightscale_tag', branch: 'v1.1.0'
+cookbook 'machine_tag',github: 'rightscale-cookbooks/machine_tag', branch: 'v1.1.0'
+cookbook 'ephemeral_lvm',github:'rightscale-cookbooks/ephemeral_lvm', branch: 'v1.0.12'
+cookbook 'ohai', '= 2.1.0'
```
Comment (cdwilhelm) [inline Berksfile]: does rs-mysql or mysql need ohai?  if it's mysql, then it should be added there.

File: metadata.rb (hunk 1 above)
Comment (rshade) [inline metadata.rb]: A blank install without that line gets 3.3.3 which leads to errors on compile. It's a hard limit on this cookbook so added here

File: Berksfile (hunk 2 above)
Comment (rshade) [inline Berksfile]: one of the dependent cookbooks has an openended ohai > 0.0.0 so needs to be pinned here

File: metadata.rb (hunk 1 above)
Comment (cdwilhelm) [inline metadata.rb]: I think this may need to go into kabam-rs-mysql not rs-mysql.  I don't think rs-mysql has any dependency on aws.

Comment (rshade) [inline metadata.rb]: database (1.5.2)
   aws (>= 0.0.0)
depends on open aws

**Pull request merged**